src/fontMetrics.js

```diff
@@ -18,30 +18,28 @@ var cjkRegex = require("./unicodeRegexes").cjkRegex;
 // dimensions, one for each of textstyle, scriptstyle, and scriptscriptstyle,
 // but we only use the textstyle ones, and scale certain dimensions accordingly.
 // See the TeXbook, page 441.
-var sigma1 = 0.025;
-var sigma2 = 0;
-var sigma3 = 0;
-var sigma4 = 0;
-var sigma5 = 0.431;
-var sigma6 = 1;
-var sigma7 = 0;
-var sigma8 = 0.677;
-var sigma9 = 0.394;
-var sigma10 = 0.444;
-var sigma11 = 0.686;
-var sigma12 = 0.345;
-var sigma13 = 0.413;
-var sigma14 = 0.363;
-var sigma15 = 0.289;
-var sigma16 = 0.150;
-var sigma17 = 0.247;
-var sigma18 = 0.386;
-var sigma19 = 0.050;
-var sigma20 = 2.390;
-var sigma21 = 1.01;
-var sigma21Script = 0.81;
-var sigma21ScriptScript = 0.71;
-var sigma22 = 0.250;
+var sigma1 = 0.025;                  // slant
+var sigma2 = 0;                      // space
+var sigma3 = 0;                      // stretch
+var sigma4 = 0;                      // shrink
+var sigma5 = 0.431;                  // xheight
+var sigma6 = [1.000, 1.171, 1.472];  // quad
+var sigma7 = 0;                      // extraspace
+var sigma8 = [0.677, 0.732, 0.925];  // num1
+var sigma9 = [0.394, 0.384, 0.387];  // num2
+var sigma10 = [0.444, 0.471, 0.504]; // num3
+var sigma11 = [0.686, 0.752, 1.025]; // denom1
+var sigma12 = [0.345, 0.344, 0.532]; // denom2
+var sigma13 = [0.413, 0.503, 0.504]; // sup1
+var sigma14 = [0.363, 0.431, 0.404]; // sup2
+var sigma15 = [0.289, 0.286, 0.294]; // sup3
+var sigma16 = [0.150, 0.143, 0.200]; // sub1
+var sigma17 = [0.247, 0.286, 0.400]; // sub2
+var sigma18 = [0.386, 0.353, 0.494]; // supdrop
+var sigma19 = [0.050, 0.071, 0.100]; // subdrop
+var sigma20 = [2.390, 1.700, 1.980]; // delim1
+var sigma21 = [1.010, 1.157, 1.420]; // delim2
```

Comment:
The values for `sigma21` that were here before had premultiplied the style's `fontSizeMultiplier`.  Since our inline styles use `em`s to position thing, I would assume that no scaling would be required because `em`s are relative to the font-size of the element.